### ISSUE TITLE
Add Distributed Systems course

### ISSUE THREAD

**Issue by atkinew0** [ossu/computer-science#821]
I have found MIT's 6.824 course on Distributed Systems  [https://pdos.csail.mit.edu/6.824/](https://pdos.csail.mit.edu/6.824/) to be a very high quality unique resource for learning about distributed systems.  Distributed Systems is a Knowledge Area in 2013CS guidelines so perhaps this should be included under the Advanced section, although I'm not quite sure where since the Systems section is currently focused on low level systems.

**Comment by waciumawanjohi** [comment]
@atkinew0 
Have you seen this issue: https://github.com/ossu/computer-science/issues/721 ? I'm curious about your thoughts after reading through that thread.

**Comment by waciumawanjohi** [comment]
As it's been 2 weeks without a response, closing this issue. Please feel free to reopen if you have more to add.

**Comment by atkinew0** [comment]
Sorry I missed your message. I can add to the discussion in #721 that I am just over halfway through the course and I have found it a very valuable learning experience.

 I have no particular distributed systems background, and keeping in mind it is a graduate level course, the homework assignments are difficult but doable. This is not one of those online courses that lead to frustration because the autograder/assignment instructors/ or whatever are confusing or out of date. In fact there is a lot of very helpful handholding with regards to the design and implementation on the labs. On the other hand I tried the old C++ assignments from the 2013 version and those homework assignments are much more obscure without having seen the relevant lectures.
